test/SILOptimizer/experimental-swift-based-closure-specialization/closure_specialize_dynamic_self.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %target-swift-frontend -emit-sil -O -primary-file %s
```

Comment:
Don't you need `-experimental-swift-based-closure-specialization` here?

Reply:
Missed by mistake. Added.